### ISSUE TITLE
train_model.py里面代码有点疑惑

### ISSUE THREAD

**Issue by abcabc3210** [naru-project/naru#4]
if xbhat.shape == xb.shape:
            if mean:
                xb = (xb * std) + mean
            loss = F.binary_cross_entropy_with_logits(
                xbhat, xb, size_average=False) / xbhat.size()[0]
mean和std没有定义
